modules/hivemq-edge-module-modbus/src/test/java/com/hivemq/edge/adapters/modbus/ModbusProtocolAdapterTest.java

```diff
@@ -63,13 +60,10 @@ void test_mergedSamples() {
 
     protected static ModBusData createSampleData() {
         final ModbusToMqttMapping pollingContext = new ModbusToMqttMapping("topic",
-                2,
+                2, "tag1",
```

Comment:
Can we not mix multiline vs single line method chains?
Reading wise it's faster and requires less effort to either read from left to right or from top to bottom.
Mixing them like this makes reading more teadious especially when just quickly parsing code.

Reply:
ah my bad, disabled auto format before commit because it breaks the OpenApi